root.tf

```diff
@@ -263,4 +264,14 @@ module "file_format_sqs_queue" {
   function       = "file-format"
   sns_topic_arns = [module.dirty_upload_sns_topic.sns_arn]
   sqs_policy     = "sns_topic"
-}
\ No newline at end of file
+}
+
+
+module "api_update_antivirus_lambda" {
```

Comment:
Is this lambda just for the antivirus, or will it be used by all the backend checks?

Reply:
Just fo the antivirus, I tried to get a single lambda to do all of the api updates and it's a bit of a nightmare in terms of parsing json so it's simpler to have one each in a multi project build.

Reply:
Do you want me to have a go at the parsing, if you think it's worth pursuing? Or do you want to try out having separate queues and lambdas and see how it goes?

Reply:
Separate queues is the way to go. It can be done, I could have got the parsing working but I think this is the cleaner way. We can always merge them later if it becomes difficult.